README.md

```diff
@@ -154,12 +154,23 @@ Temporal::Worker.run(worker_1, worker_2, worker_3)
 Please note that similar to `Temporal::Worker#run`, `Temporal::Worker.run` accepts a block that
 behaves the same way — the workers will be shut down when the block finishes.
 
+You can also configure your worker to listen on process signals to initiate a shutdown:
+
+```ruby
+Temporal::Worker.run(worker_1, worker_2, worker_3, stop_on_signal: %w[INT TERM])
```

Comment:
I'm debating whether you should call this `shutdown_signals` because we refer to this concept as shutdown everywhere else but here.

Reply:
I actually like the `shutdown_signals`, makes a lot of sense